.github/workflows/build.yaml

```diff
@@ -9,10 +9,18 @@ jobs:
     runs-on: ubuntu-latest
     container: debian:bookworm
     steps:
+      - run: mkdir -p catkin_ws/src
       - name: Checkout Project
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          path: catkin_ws/src/aioros
       - run: apt update
       - name: Install dependencies
-        run: apt install -y python3-anyio python3-h11 python3-httpx python3-lxml python3-pytest python3-pytest-cov python3-pytest-mock python3-genpy python3-std-msgs python3-std-srvs python3-rosgraph-msgs python3-actionlib-msgs
-      - name: Run tests
-        run: PYTHONPATH=. pytest-3 -s --cov=aioros
+        run: apt install -y python3-anyio python3-h11 python3-httpx python3-lxml python3-pytest python3-pytest-cov python3-pytest-mock python3-genpy python3-std-msgs python3-std-srvs python3-rosgraph-msgs python3-actionlib-msgs catkin build-essential
```

Comment:
maybe don't use `build-essential`?

Reply:
what do you recommend instead?
I am still adding support for pytests in cmake…

Reply:
g++, I guess

Reply:
(I think you can teach cmake not to depend on it, not sure if it works for catkin)

Reply:
I'm creating this PR in preparation of the pybind tf2 wrapper, which means I will need a compiler soon ;)

Reply:
ok. I think of build-essential as a Debian thing of the past which is hard to get rid of, not sure we should expose that to ROS, that's why I would propose to rather declare what you actually need.

Reply:
done